resources/public/api/conf/1.0/examples/number-of-employees-response.json

```diff
@@ -4,31 +4,40 @@
       "href": "/organisations/details/number-of-employees?matchId=57072660-1df9-4aeb-b4ea-cd2d7f96e430"
     }
   },
-  "payeReference": "RT882d/456",
-  "counts": [
+  "employeeCounts": [
     {
-      "dateOfCount": "2019-03",
-      "numberOfEmployees": 1234
-    },
-    {
-      "dateOfCount": "2019-11",
-      "numberOfEmployees": 1466
-    },
-    {
-      "dateOfCount": "2019-12",
-      "numberOfEmployees": 1765
-    },
-    {
-      "dateOfCount": "2020-01",
-      "numberOfEmployees": 1666
-    },
-    {
-      "dateOfCount": "2020-02",
-      "numberOfEmployees": 1589
-    },
-    {
-      "dateOfCount": "2020-03",
-      "numberOfEmployees": 1555
+      "payeReference": "RT882d/456",
```

Comment:
think there's been a duplication here of counts within counts/nested payeReference?